style.css

```diff
@@ -0,0 +1,511 @@
+  margin-top: 30px;
+  align-items: center;
+}
+
+.recent_work_container img {
+  width: 280px;
+}
+
+.recent_work_container_text {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  justify-items: center;
+  margin-left: 20px;
+  margin-top: 10px;
+  gap: 10px;
+}
+
+.recent_work_container_text h3 {
+  font-family: 'Crete Round', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 32px;
+  line-height: 44px;
+  display: flex;
+  align-items: center;
+  color: #091e42;
+}
+
+.recent_work_container_text p {
+  width: 300px;
+  height: 147px;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  color: #091e42;
+}
+
+.butons_container {
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+  gap: 10px;
+}
+
+.butons_container button {
+  text-align: center;
+  background: none;
+  height: 40px;
+  width: 60px;
+  border: 1px solid rgb(61, 59, 59);
+  position: relative;
+  left: -17px;
+  background-color: aliceblue;
+}
+
+.recent_work_container .project_btn {
+  text-align: center;
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  padding: 12px;
+  gap: 10px;
+  width: 124px;
+  height: 48px;
+  background: #ff6b00;
+  border: none;
+  font-weight: bold;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  color: #fff;
+}
+
+.container {
+  display: grid;
+  grid-template-columns: repeat(1, 1fr);
+  grid-template-rows: repeat(6, 1fr);
+  margin-top: 20px;
+  overflow: hidden;
+}
+
+.box {
+  padding: 23px;
+  background-image: url('./images/placeholder.png');
+  width: 240px;
+  height: 300px;
+  background-repeat: no-repeat;
+  background-size: cover;
+  overflow: hidden;
+  margin-left: 20px;
+}
+
+.box h4 {
+  width: 295px;
+  height: 81px;
+  font-family: 'Crete Round', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 32px;
+  line-height: 44px;
+  color: #fff;
+}
+
+.box p {
+  width: 200px;
+  height: 99px;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  color: #fff;
+}
+
+.grid button {
+  background: rgba(255, 255, 255, 0.24);
+  width: 80px;
+  color: #fff;
+  font-weight: bold;
+  position: relative;
+  top: 50px;
+  left: 1.4px;
+}
+
+.btn {
+  background-color: #ff6b00;
+  width: 300px;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 17px;
+  text-align: center;
+  align-items: center;
+  color: #fff;
+  border: none;
+  height: 50px;
+  position: relative;
+  top: 55px;
+  right: 22px;
+}
+
+.btn:hover {
+  background-color: #1c1a19;
+  color: white;
+}
+
+.second {
+  position: relative;
+  bottom: 200px;
+}
+
+.about_container {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  justify-items: center;
+  align-items: flex-start;
+  background-color: #1c1a19;
+  width: 320px;
+  height: 100%;
+  margin-top: 40px;
+}
```

Comment:
- [ ] As mentioned by the previous [reviewer](https://github.com/malodev1234/Portfolio_setup_mobile_first/pull/3#discussion_r1155082029), I suggest you mind your  design in this case you gave your section a definite/non-responsive width wich gives it extra white space on the right, I suggest you don't give it a width 😉 

![space](https://user-images.githubusercontent.com/102636489/229304954-5104faee-4482-44a9-a984-c0e27e4b06dc.png)